packages/docs/src/components/doc/Parameters.vue

```diff
@@ -13,9 +13,17 @@
       sort-by="name"
       :items-per-page="-1"
       class="component-parameters pa-2"
+      hide-default-footer
     >
-      <template v-slot:item="{ item }">
-        <v-flex xs12 grey lighten-2 mt-2>
+      <template v-slot:default="{ items }">
```

Comment:
```suggestion
      <template #default="{ items }">
```

Reply:
What's your rationale for adding the shorthand _only_ to the default slot? My thinking with not using the shorthand was to not confuse users that might not know about it.